app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def show; end
+end
```

Comment:
- [ ] Kindly add the `index` action as you have done in the `posts controller`
- [ ] Remember to add a corresponding view for the index action.